build-system/global-configs/README.md

```diff
@@ -9,7 +9,8 @@ besides 1 or 0.
 
 - `name`: Experiment name
 - `environment`: Specify the type of environment the experiment runs. Only support `AMP` and `INABOX`.
-- `command`: Command used to build the experiment
+- `command`: Command used to build the experiment's nomodule build
+- `esmCommand`: Command used to build the experiment's module build
```

Comment:
Since some people probably don't know what to put here like me, I wonder if it is best to auto generate the `esmCommand`?

Reply:
I think it's better to be explicit about this, but the doc could add "this is usually the same as `command` with `--esm` added to the `gulp dist` command"

Reply:
Sounds good. One more question about e2e test. We build the testing binary by replacing `gulp dist` with `gulp dist --fortesting` in `command`. Do we run e2e test on the esm build as well? If so I think we need to apply the same trick on `esmCommand` as well. cc @estherkim